tests/observability/metrics/conftest.py

```diff
@@ -891,6 +894,22 @@ def vm_virt_launcher_pod_requested_memory(vm_for_test):
     )
 
 
+@pytest.fixture()
+def generated_api_deprecated_requests(prometheus):
+    initial_metric_value = int(
+        get_metrics_value(
+            prometheus=prometheus,
+            metrics_name=KUBEVIRT_API_REQUEST_DEPRECATED_TOTAL_WITH_VERSION_AND_RESOURCE,
+        )
+    )
+    for oc_command in range(COUNT_FIVE):
+        run_command(
+            command=shlex.split("oc get --raw /apis/kubevirt.io/v1alpha3/namespaces/default/virtualmachines"),
```

Comment:
Please don't use oc command directly. Use VirtualMachine from ocp_resources.virtual_machine instead. You can set api_version to a older one and do a get call to get list of vms